git-branchless/src/commands/move.rs

```diff
@@ -142,18 +142,6 @@ pub fn r#move(
                     return Ok(ExitCode(1));
                 };
 
-                let component_heads = dag.query().heads(component.clone())?;
-                if component_heads.count()? != 1 {
-                    writeln!(
-                        effects.get_error_stream(),
-                        "The --exact flag can only be used to move ranges with exactly 1 head.\n\
-                         Received range with {} heads: {:?}",
-                        component_heads.count()?,
-                        component_heads
-                    )?;
-                    return Ok(ExitCode(1));
-                };
-
```

Comment:
It's worth noting that `--insert` still has a 1 HEAD restriction, but that's already handled on line 364 and it's not unique to `--exact` (ie it also applies to subtrees).